orttraining/orttraining/core/framework/ortmodule_graph_builder.cc

```diff
@@ -63,6 +63,33 @@ Status OrtModuleGraphBuilder::Initialize(std::istream& model_istream,
   return Status::OK();
 }
 
+// Inline functions that do not have a gradient builder registered
+Status OrtModuleGraphBuilder::InlineFunctions(Graph& graph, bool& modified_graph) const {
+  // recurse into nested graphs first so we process from bottom up
```

Comment:
i actually even not sure does ORT training support subgraph today. do you need that support?